packages/kamino-sdk/src/Kamino.ts

```diff
@@ -282,6 +284,50 @@ export class Kamino {
     }
   }
 
+  /**
+   * Get amount of specified token in all Kamino live strategies
+   * @param tokenMint token mint pubkey
+   */
+  async getTotalTokensInStrategies(tokenMint: PublicKey | string): Promise<TotalStrategyVaultTokens> {
+    // TODO: update and use this method call to only fetch live strategies once strategy status on-chain data is ready
+    // const strategies = await this.getAllStrategiesWithFilters({
+    //   strategyType: undefined,
+    //   strategyCreationStatus: undefined,
+    // });
+    const strategies = await this.getStrategies(this._config.kamino.liveStrategies.map((x) => x.address));
+    let totalTokenAmount = new Decimal(0);
+    const vaults: StrategyVaultTokens[] = [];
+    for (const liveStrategy of this._config.kamino.liveStrategies) {
+      const strategy = strategies.find(
+        (x) =>
+          x?.sharesMint.toString() === liveStrategy.shareMint.toString() &&
```

Comment:
sharemint is unique, you can just filter on that or you can just make a hashmap higher up

Reply:
I need to find the strategy pubkey to include in the response, that's why I am searching for the strategy like this.

Reply:
This is used to find the strategy onchain data which I pass into the get strategy balance function (faster if I pass onchain data instead of strategy pubkey so it doesnt refetch strategy onchain again)